### ISSUE TITLE
Change aria-controls on mobile nav buttons to point at nav list id instead of main nav id

### ISSUE THREAD

**Issue by ruuen** [ruuendigital/challenge-news-homepage#17]
Add an ID to the nav list element of `nav-list`, and change the aria-controls values on both mobile nav buttons to reference this `nav-list` ID.

Id against the main nav element can be removed as this is no longer referenced elsewhere.

Grace @ Frontend Mentor had the below feedback:
> The aria-controls is pointing to an ID on the wrong element. It should be pointing to an ID on the actual content it is toggling (the UL)
